src/monodroid/monodroid.targets

```diff
@@ -8,7 +9,16 @@
   <ItemGroup>
     <CFiles Include="jni\**\*.c" />
   </ItemGroup>
+  <Target Name="_GetRuntimes">
+    <ValueToItems
```

Comment:
Should it be `SplitScalarIntoItems` or `ScalarToItems`? Though, maybe `Value` also kinda says the same thing.